groups/bsl/bslstl/bslstl_optional.h

```diff
@@ -12733,7 +12746,7 @@ bool operator!=(const bsl::optional<LHS_TYPE>& lhs,
                 const std::optional<RHS_TYPE>& rhs)
 {
     if (lhs && rhs) {
-        return lhs.value() != rhs.value();
+        return *lhs != *rhs;
```

Comment:
Can we please make the line above this one

```
if (lhs.has_value() && rhs.has_value())
```

for consistency with bsl-to-bsl comparisons?

I counted 8 such changes including changes to places like `return !lhs || *lhs < *rhs;` (changing to `return !lhs.has_value() ||  *lhs < *rhs;`)

Reply:
done